### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by Falci** [ordinals/ord#3483] [Q&A]
Is this legit?
https://hub.docker.com/r/ordinals/ord

Couldn't find any reference for a ready-to-use image. Just checking.

**Comment by raphjaph** [comment]
I might have reserved that but not sure. You can just build the docker image by cloning repo and then `docker build -t ord .`

**Reply by Falci** [reply]
Thanks.

I added a [GitHub Action](https://github.com/Falci/ord/blob/master/.github/workflows/docker.yml) to build and sent to hub.docker.com.

Maybe we could have an official version?
